packages/wallets/src/components/WalletHistory.tsx

```diff
@@ -141,45 +180,9 @@ const getCols = (type: WalletType, isSyncing, getOfferRecord, navigate) => [
     },
     title: <Trans>To</Trans>,
   },
-  {
-    field: (row: Row) => (
-      <Typography color="textSecondary" variant="body2">
-        {moment(row.createdAtTime * 1000).format('LLL')}
-      </Typography>
-    ),
-    title: <Trans>Date</Trans>,
-  },
-  {
-    field: (row: Row, metadata) => {
-      const isOutgoing = [TransactionType.OUTGOING, TransactionType.OUTGOING_TRADE].includes(row.type);
 
-      return (
-        <>
-          <strong>{isOutgoing ? <Trans>-</Trans> : <Trans>+</Trans>}</strong>
-          &nbsp;
-          <strong>
-            <FormatLargeNumber value={type === WalletType.CAT ? mojoToCAT(row.amount) : mojoToChia(row.amount)} />
-          </strong>
-          &nbsp;
-          {metadata.unit}
-        </>
-      );
-    },
-    title: <Trans>Amount</Trans>,
-  },
-  {
-    field: (row: Row, metadata) => (
-      <>
-        <strong>
-          <FormatLargeNumber value={mojoToChia(row.feeAmount)} />
-        </strong>
-        &nbsp;
-        {metadata.feeUnit}
-      </>
-    ),
-    title: <Trans>Fee</Trans>,
-  },
   {
+    width: '60px',
```

Comment:
When hovering over the expand/collapse button, the highlighted circle is cut off

```suggestion
    width: '66px',
```